src/OnchainKitConfig.ts

```diff
@@ -25,5 +27,13 @@ export const getOnchainKitConfig = <K extends keyof typeof ONCHAIN_KIT_CONFIG>(
  */
 export const setOnchainKitConfig = (properties: SetOnchainKitConfig) => {
   Object.assign(ONCHAIN_KIT_CONFIG, properties);
+  if (!ONCHAIN_KIT_CONFIG.apiKey) {
```

Comment:
wrong assumption, left a comment after on this.

Reply:
removed mandatory api key